src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComInterfaceGenerator.cs

```diff
@@ -637,24 +637,15 @@ private static ClassDeclarationSyntax GenerateInterfaceInformation(ComInterfaceI
 
             static ExpressionSyntax CreateEmbeddedDataBlobCreationStatement(ReadOnlySpan<byte> bytes)
             {
-                var literals = new LiteralExpressionSyntax[bytes.Length];
+                var literals = new CollectionElementSyntax[bytes.Length];
```

Comment:
Does this require language version guard?

Reply:
Generally, we haven't been adding language version guards in the interop source generators. It's technically more correct to do so, but I'd rather hold off until we have a customer who explicitly downgrades the language version and files an issue with us before I add them.